src/website/views/components/steps/_examples.pug

```diff
@@ -121,27 +150,442 @@ p.-text
       <ul class="chi-steps -horizontal-label">
         <li class="-completed">
           <div class="chi-steps__icon"></div>
-          <a href="#">Step 1</a>
+          <div class="chi-steps__content">
+            <a href="#">Step 1</a>
+          </div>
           <div class="chi-steps__line"></div>
         </li>
         <li class="-completed">
           <div class="chi-steps__icon"></div>
-          <a href="#">Step 2</a>
+          <div class="chi-steps__content">
+            <a href="#">Step 2</a>
+          </div>
           <div class="chi-steps__line"></div>
         </li>
         <li class="-active">
           <div class="chi-steps__icon"></div>
-          <a href="#">Step 3</a>
+          <div class="chi-steps__content">
+            <a href="#">Step 3</a>
+          </div>
           <div class="chi-steps__line"></div>
         </li>
         <li>
           <div class="chi-steps__icon"></div>
-          <a href="#">Step 4</a>
+          <div class="chi-steps__content">
+            <a href="#">Step 4</a>
+          </div>
           <div class="chi-steps__line"></div>
         </li>
         <li>
           <div class="chi-steps__icon"></div>
-          <a href="#">Step 5</a>
+          <div class="chi-steps__content">
+            <a href="#">Step 5</a>
+          </div>
+        </li>
+      </ul>
+
+h3 Vertical
+p.-text
+  | To display steps vertically, apply the class <code>-vertical</code> to the steps container.
```

Comment:
```suggestion
  | To display steps vertically, apply the class <code>-vertical</code> to the container.
```
Thoughts on putting it this way? Just to avoid repetition of the work `steps`